scripts/winesapos-convert.sh

```diff
@@ -96,6 +90,13 @@ Include = /etc/pacman.d/chaotic-mirrorlist" | sudo tee -a /etc/pacman.conf
     # shellcheck disable=SC2103
     cd ..
     sudo rm -rf yay
+    # GuestSneezeOSDev: add balenaEtcher
+    git clone https://aur.archlinux.org/balena-etcher.git
```

Comment:
Instead of installing the AUR package, use this code which mirrors how we install the AppImage. I updated it to work with the conversion script.

```
export ETCHER_VER="1.19.21"
curl --location "https://github.com/balena-io/etcher/releases/download/v${ETCHER_VER}/balenaEtcher-${ETCHER_VER}-x64.AppImage" --output /home/"${USER}"/Desktop/balenaEtcher.AppImage
chmod +x /home/"${USER}"/Desktop/balenaEtcher.AppImage
```

Reply:
okay